WireAPI/Tests/WireAPITests/APIs/TeamsAPI/TeamsAPITests.swift

```diff
@@ -249,53 +249,59 @@ final class TeamsAPITests: XCTestCase {
         }
     }
 
-    func testGetLegalholdStatus_SuccessResponse_200_V0() async throws {
+    func testGetLegalhold_SuccessResponse_200_V0() async throws {
         // Given
         let httpClient = try HTTPClientMock(
             code: .ok,
             jsonResponse: """
             {
-                "status": "pending"
+                "status": "pending",
+                "last_prekey": {
+                    "id": 12345,
+                    "key": "foo"
+                }
             }
             """
         )
 
         let sut = TeamsAPIV0(httpClient: httpClient)
 
         // When
-        let result = try await sut.getLegalholdStatus(
+        let result = try await sut.getLegalhold(
             for: Team.ID(),
             userID: UUID()
         )
 
         // Then
-        XCTAssertEqual(result, .pending)
+        let expectedPrekey = LegalHoldPrekey(id: 12_345, base64EncodedKey: "foo")
+        XCTAssertEqual(result.status, .pending)
+        XCTAssertEqual(result.prekey, expectedPrekey)
     }
 
-    func testGetLegalholdStatus_FailureResponse_InvalidRequest_V0() async throws {
+    func testGetLegalhold_FailureResponse_InvalidRequest_V0() async throws {
```

Comment:
we could avoid repetition with a method:

```
func internalTest_GetLegalhold_Failure(expectedError: Error, errorLabel: String, file: StaticString = #file, line: UInt = #line) {
         // Given
        let httpClient = try HTTPClientMock(code: .notFound, errorLabel: errorLabel)
        let sut = TeamsAPIV0(httpClient: httpClient)

        // Then
        await XCTAssertThrowsError(expectedError) {
            // When
            try await sut.getLegalhold(
                for: Team.ID(),
                userID: UUID()
            )
        }
}
```

Reply:
done in https://github.com/wireapp/wire-ios/pull/1999/commits/6808852586237e2f91c091ee3fd733489b1fb767